### PR TITLE
Introduce NodeToNodeV_14

### PR DESCRIPTION
**Pull request by crocodile-dentist**
# Description

No changes in this version, it will be used to identify adoption rate of Chang +1 HF.

# Checklist

### Quality
* [x] Commit sequence makes sense and have useful messages, see [ref][contrib#git-history].
* [ ] New tests are added and existing tests are updated.
* [ ] Self-reviewed the PR.

### Maintenance
* [ ] Linked an [issue][link-issue] or added the PR to the current sprint of [`ouroboros-network`][project] project.
* [ ] Added labels.
* [x] Updated changelog files.
* [ ] The documentation has been properly updated, see [ref][contrib#documentation].

[project]: https://github.com/orgs/IntersectMBO/projects/5/views/1
[link-issue]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword=
[contrib#git-history]: https://github.com/IntersectMBO/ouroboros-network/blob/master/CONTRIBUTING.md#git-history
[contrib#documentation]: https://github.com/IntersectMBO/ouroboros-network/blob/master/CONTRIBUTING.md#documentation

### CODE REVIEW

File: docs/network-spec/miniprotocols.tex [hunk 1]
```diff
@@ -1494,21 +1494,10 @@ \section{Node-to-node protocol}
...
-      $NodeToNodeV\_3*$ & introduction of keep-alive mini-protocol \\\hline
-      $NodeToNodeV\_4*$ & introduction of diffusion mode in handshake mini-protocol \\\hline
-      $NodeToNodeV\_5*$ & \\\hline
-      $NodeToNodeV\_6*$ & transaction submission version 2 \\\hline
-      $NodeToNodeV\_7$ & new keep-alive, Alonzo ledger era \\\hline
-      $NodeToNodeV\_8$ & chain-sync \& block-fetch pipelining \\\hline
-      $NodeToNodeV\_9$ & Babbage ledger era \\\hline
-      $NodeToNodeV\_10$ & Full duplex connections \\\hline
-      $NodeToNodeV\_11$ & Peer sharing willingness \\\hline
-      $NodeToNodeV\_12$ & No observable changes \\\hline
```
Comment (coot) [inline docs/network-spec/miniprotocols.tex:1508]: We should either move them to a separate section or record the very early versions in the code (this list is more complete than what we have in the code).  The former looks better to me.

File: docs/network-spec/miniprotocols.tex [hunk 2]
```diff
@@ -1530,14 +1519,6 @@ \section{Node-to-client protocol}
...
     \begin{tabular}{l|l}
       version & description \\\hline\hline
-      $NodeToClientV\_1*$ & initial version\\\hline
-      $NodeToClientV\_2*$ & added local-query mini-protocol\\\hline
-      $NodeToClientV\_3*$ & \\\hline
-      $NodeToClientV\_4*$ & new queries added to local state query mini-protocol\\\hline
-      $NodeToClientV\_5*$ & allegra \\\hline
-      $NodeToClientV\_6*$ & mary \\\hline
-      $NodeToClientV\_7*$ & new queries added to local state query mini-protocol\\\hline
-      $NodeToClientV\_8*$ & codec changed for local state query mini-protocol\\\hline
```
Comment (coot) [inline docs/network-spec/miniprotocols.tex:1540]: As above.

File: docs/network-spec/miniprotocols.tex (hunk 1 above)
Comment (crocodile-dentist) [reply docs/network-spec/miniprotocols.tex:1508]: I've moved them to a new appendix chapter.

File: docs/network-spec/miniprotocols.tex (hunk 2 above)
Comment (crocodile-dentist) [reply docs/network-spec/miniprotocols.tex:1540]: Moved to appendix.

**Review by karknu — CHANGES_REQUESTED** [review]
LGTM, but you should add support for V14 to cardano-ping too.

**Review by karknu — APPROVED** [review]
LGTM

**Review by coot — APPROVED** [review]
LGTM

**Pull request merged**